modules/hooks.nix

```diff
@@ -1638,7 +1640,7 @@ in
     };
 
```

Comment:
Ah, okay, we're hitting the limitations of the freetype+submodule system. We should manually add a warning to help people switch over to biome. Something like this:

```suggestion

  config.warnings =
    lib.optional cfg.hooks.rome.enable ''
      The hook `hooks.rome` has been renamed to `hooks.biome`.
    '';

```

Reply:
I like this approach :+1: